### ISSUE TITLE
Leaked file descriptors

### ISSUE THREAD

**Issue by tasleson** [rhinstaller/libblockdev#204]
These show up all over in the logs.

```
Verify that it is possible to wipe an ext2 file system ... File descriptor 6 (pipe:[2436816]) leaked on pvcreate invocation. Parent PID 22887: sh
File descriptor 7 (pipe:[2441744]) leaked on pvcreate invocation. Parent PID 22887: sh
File descriptor 9 (pipe:[2446707]) leaked on pvcreate invocation. Parent PID 22887: sh
File descriptor 11 (pipe:[2451810]) leaked on pvcreate invocation. Parent PID 22887: sh
File descriptor 13 (pipe:[2456933]) leaked on pvcreate invocation. Parent PID 22887: sh
File descriptor 15 (pipe:[2462100]) leaked on pvcreate invocation. Parent PID 22887: sh
ok
test_ext3_wipe (fs_test.ExtTestWipe)
Verify that it is possible to wipe an ext3 file system ... File descriptor 6 (pipe:[2436816]) leaked on pvcreate invocation. Parent PID 23009: sh
File descriptor 7 (pipe:[2441744]) leaked on pvcreate invocation. Parent PID 23009: sh
File descriptor 9 (pipe:[2446707]) leaked on pvcreate invocation. Parent PID 23009: sh
File descriptor 11 (pipe:[2451810]) leaked on pvcreate invocation. Parent PID 23009: sh
File descriptor 13 (pipe:[2456933]) leaked on pvcreate invocation. Parent PID 23009: sh
File descriptor 15 (pipe:[2462100]) leaked on pvcreate invocation. Parent PID 23009: sh
ok
test_ext4_wipe (fs_test.ExtTestWipe)
Verify that it is possible to wipe an ext4 file system ... File descriptor 6 (pipe:[2436816]) leaked on pvcreate invocation. Parent PID 23131: sh
File descriptor 7 (pipe:[2441744]) leaked on pvcreate invocation. Parent PID 23131: sh
File descriptor 9 (pipe:[2446707]) leaked on pvcreate invocation. Parent PID 23131: sh
File descriptor 11 (pipe:[2451810]) leaked on pvcreate invocation. Parent PID 23131: sh
File descriptor 13 (pipe:[2456933]) leaked on pvcreate invocation. Parent PID 23131: sh
File descriptor 15 (pipe:[2462100]) leaked on pvcreate invocation. Parent PID 23131: sh
```

**Comment by vpodzime** [comment]
No idea where these are coming from neither I care about them (in tests). Feel free to find it out, but I don't consider this an issue.

**Comment by tasleson** [comment]
Addressed with https://github.com/rhinstaller/libblockdev/pull/206

**Comment by vpodzime** [comment]
> Addressed with #206

Thanks :)
